user/user.go

```diff
@@ -81,6 +81,12 @@ func User(w http.ResponseWriter, r *http.Request) {
 				Method:  http.MethodPatch,
 				Roles:   []string{models.UserRoleAdmin},
 			},
+			{
+				Route:   "/user/v1",
```

Comment:
The domain already has user, so you can leave only `v1`
```suggestion
				Route:   "/v1",
```